Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadInstanceBaker.cs

```diff
@@ -405,7 +259,7 @@ private Matrix3d GetMatrix3d(Matrix4x4 matrix, string units)
   }
 
   // https://forums.autodesk.com/t5/net/set-blocktransform-values/m-p/6452121#M49479
-  private static double[] MakePerpendicular(Matrix3d matrix)
+  private double[] MakePerpendicular(Matrix3d matrix)
```

Comment:
Why un-static this?

Reply:
why not? it was unnecessarily static that only called in a function that is not static